src/components/header.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { HeadStyle } from "../styled/style";
+import "./style.css";
+
+type ScoreProps = {
+  score: number;
+  bestScore: number;
+};
+
+const UserHeader: React.FC<ScoreProps> = ({ score, bestScore }) => {
+  return (
+    <>
```

Comment:
fragment not needed in case of a single child. fragments are used to house 2 or more parallel children

Reply:
Resolved & Noted. Will avoid in future